Gemfile

```diff
@@ -4,11 +4,3 @@ source "https://rubygems.org"
 ruby "2.4.1"
 
 gemspec path: "."
-gemspec path: "decidim-core"
-gemspec path: "decidim-system"
-gemspec path: "decidim-admin"
-gemspec path: "decidim-dev"
-gemspec path: "decidim-api"
-gemspec path: "decidim-pages"
-gemspec path: "decidim-comments"
-gemspec path: "decidim-meetings"
```

Comment:
With this, the lock file uses the released version of the gems. Is that what you want?

In case it helps/makes-sense, I was getting some annoying warnings with the previous "multiple gemspec setup" and fixed them locally with

```patch
commit cd5537596e2a969ed1086a7f6fb2602e816e69fe
Author: David Rodríguez <deivid.rodriguez@riseup.net>
Date:   Tue Apr 25 13:47:47 2017 -0300

    Fix bundler warnings
    
    ```
    Your Gemfile lists the gem decidim-dev (= 0.0.8) more than once.
    You should probably keep only one of them.
    While it's not a problem now, it could cause errors if you change the version of one of them later.
    Your Gemfile lists the gem decidim-proposals (= 0.0.8) more than once.
    You should probably keep only one of them.
    While it's not a problem now, it could cause errors if you change the version of one of them later.
    Your Gemfile lists the gem decidim-results (= 0.0.8) more than once.
    You should probably keep only one of them.
    While it's not a problem now, it could cause errors if you change the version of one of them later.
    ```

diff --git a/Gemfile b/Gemfile
index b46713fd..29ee0d21 100644
--- a/Gemfile
+++ b/Gemfile
@@ -4,11 +4,11 @@ source "https://rubygems.org"
 ruby "2.4.1"
 
 gemspec path: "."
-gemspec path: "decidim-core"
-gemspec path: "decidim-system"
-gemspec path: "decidim-admin"
-gemspec path: "decidim-dev"
-gemspec path: "decidim-api"
-gemspec path: "decidim-pages"
-gemspec path: "decidim-comments"
-gemspec path: "decidim-meetings"
+gem "decidim-core", path: "decidim-core"
+gem "decidim-system", path: "decidim-system"
+gem "decidim-admin", path: "decidim-admin"
+gem "decidim-dev", path: "decidim-dev"
+gem "decidim-api", path: "decidim-api"
+gem "decidim-pages", path: "decidim-pages"
+gem "decidim-comments", path: "decidim-comments"
+gem "decidim-meetings", path: "decidim-meetings"
diff --git a/Gemfile.lock b/Gemfile.lock
index 439756cc..ba5b68fd 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -589,8 +589,6 @@ DEPENDENCIES
   decidim-dev!
   decidim-meetings!
   decidim-pages!
-  decidim-proposals (= 0.0.8)
-  decidim-results (= 0.0.8)
   decidim-system!
   rake (~> 12.0.0)
   rspec (~> 3.0)
```

and

```patch
commit 431a15397bc216b01eda0b04b02138a4f8a5eae2
Author: David Rodríguez <deivid.rodriguez@riseup.net>
Date:   Tue Apr 25 13:49:22 2017 -0300

    Add a couple of missing modules to Gemfile

diff --git a/Gemfile b/Gemfile
index 29ee0d21..102eef33 100644
--- a/Gemfile
+++ b/Gemfile
@@ -12,3 +12,5 @@ gem "decidim-api", path: "decidim-api"
 gem "decidim-pages", path: "decidim-pages"
 gem "decidim-comments", path: "decidim-comments"
 gem "decidim-meetings", path: "decidim-meetings"
+gem "decidim-proposals", path: "decidim-proposals"
+gem "decidim-results", path: "decidim-results"
diff --git a/Gemfile.lock b/Gemfile.lock
index ba5b68fd..907d3d07 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -20,18 +20,6 @@ PATH
       kaminari (~> 1.0.0.rc1)
       rectify (~> 0.8)
       searchlight (~> 4.1.0)
-    decidim-proposals (0.0.8)
-      decidim-comments (= 0.0.8)
-      decidim-core (= 0.0.8)
-      kaminari (~> 1.0.1)
-      rectify (~> 0.8.0)
-      social-share-button (~> 0.9.0)
-    decidim-results (0.0.8)
-      decidim-comments (= 0.0.8)
-      decidim-core (= 0.0.8)
-      kaminari (~> 1.0.0.rc1)
-      rectify (~> 0.8)
-      searchlight (~> 4.1.0)
 
 PATH
   remote: decidim-admin
@@ -150,6 +138,26 @@ PATH
       rectify (~> 0.8.0)
 
 PATH
+  remote: decidim-proposals
+  specs:
+    decidim-proposals (0.0.8)
+      decidim-comments (= 0.0.8)
+      decidim-core (= 0.0.8)
+      kaminari (~> 1.0.1)
+      rectify (~> 0.8.0)
+      social-share-button (~> 0.9.0)
+
+PATH
+  remote: decidim-results
+  specs:
+    decidim-results (0.0.8)
+      decidim-comments (= 0.0.8)
+      decidim-core (= 0.0.8)
+      kaminari (~> 1.0.0.rc1)
+      rectify (~> 0.8)
+      searchlight (~> 4.1.0)
+
+PATH
   remote: decidim-system
   specs:
     decidim-system (0.0.8)
@@ -589,6 +597,8 @@ DEPENDENCIES
   decidim-dev!
   decidim-meetings!
   decidim-pages!
+  decidim-proposals!
+  decidim-results!
   decidim-system!
   rake (~> 12.0.0)
   rspec (~> 3.0)
```